test/DataMCSpectraComparison/histosSimplified.py

```diff
@@ -187,14 +189,14 @@
             assert alldil.tight_cut == trigger_match
             alldil.tight_cut = prescaled_trigger_match
 
-    # Histos now just needs to know which leptons and dileptons to use.
-	histos = HistosFromPAT.clone(lepton_src = cms.InputTag(leptons_name, 'muons'), dilepton_src = cms.InputTag(name))
+        # Histos now just needs to know which leptons and dileptons to use.
```

Comment:
What's the reason of indentation change? Was it not working? Please elaborate

Reply:
This didn't work in the previous codes because Misha changed the indentation when I changed the code for the di-electron channel.

This is the same change Misha modified to the code again.